llvm/test/CodeGen/X86/jump-table-size-section.ll

```diff
@@ -0,0 +1,97 @@
+; RUN: llc %s -o - -emit-jump-table-sizes-section -verify-machineinstrs --relocation-model=pic | FileCheck --check-prefix=CHECK %s
+; RUN: llc %s -o - -verify-machineinstrs --relocation-model=pic | FileCheck --check-prefix=NOFLAG %s
```

Comment:
You will need explicit triples - these tests may be run where the default target is not PS5 and the default object file format is not ELF.